Sources/Subprocess/Platforms/Subprocess+Darwin.swift

```diff
@@ -81,6 +81,22 @@ public struct PlatformOptions: Sendable {
                 inout posix_spawn_file_actions_t?
             ) throws -> Void
         )? = nil
+    /// A closure to configure platform-specific
+    /// spawning constructs. This closure enables direct
+    /// configuration or override of underlying platform-specific
+    /// spawn settings that `Subprocess` utilizes internally,
+    /// in cases where Subprocess does not provide higher-level
+    /// APIs for such modifications.
+    ///
+    /// On Darwin, Subprocess uses `posix_spawn()` as the
+    /// underlying spawning mechanism, but may require an initial `fork()`
```

Comment:
Could you explicitly mention that

1) On Darwin you can set both `preExecProcessAction` and `preSpawnProcessConfigurator`, and
2) Setting `preExecProcessAction` will cause Subprocess to use the "pre-fork" and use posix_spawn solely as `exec` replacement with `POSIX_SPAWN_SETEXEC` flag.